aissemble-open-inference-protocol-examples/aissemble-oip-fastapi-auth/pyproject.toml

```
@@ -0,0 +1,35 @@
+[tool.poetry]
+packages = [{include = "aissemble-oip-fastapi-auth", from = "src"}]
+
+[project]
+name = "aissemble-oip-fastapi-auth"
+version = "1.0.0.dev"
+description = "A FastAPI Example of the Open Inference Protocol with authorization/authentication"
+authors = [
+    {name = "aiSSEMBLE Solutions Community", email = "aissemble@bah.com"}
+]
+readme = "README.md"
+requires-python = ">=3.11.4"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "License :: Other/Proprietary License",
+    "Operating System :: OS Independent",
+    "Framework :: FastAPI",
+    "Intended Audience :: Developers"
+]
+license = { text = "Booz Allen Public License v1.0" }
+license-files = ["LICENSE*"]
+dynamic = ["version", "dependencies"]
+
+[tool.poetry.dependencies]
+krausening = ">=20"
```

C: Missing dependency on aissemble-oip-fastapi. Causes errors when trying to start the app via poetry. Also, krausening is not used directly so should be removed. (It will still be pulled in by the oip module in order to read the config file.)
```suggestion
aissemble-open-inference-protocol-fastapi = {path = "../../aissemble-open-inference-protocol-fastapi", develop = true}
```

---

- [ ] Add missing dependency